style-2.css

```diff
@@ -0,0 +1,80 @@
+body{
+    margin: 0;
+}
+
+.header{
+    background-color: black;
+    color: white;
+    display: flex;
+    justify-content: space-between;
+}
+
+.site-title{
+    font-size: 3rem;
+    margin: 5px;
+}
+
+.search-bar{
+    margin: 15px;
+}
+
+.search-input{
+    padding: 5px;
+    width: 200px;
+    font-size: medium;
+}
+
+.search-btn{
+    margin-left: -5px;
+    padding: 5px;
+    width: 80px;
+    font-size: medium;
+}
+
+.main{
+    margin: 20px;
+}
+
+.container{
+    background-color: lightblue;
+}
+
+.videos-listing{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
+
+.video{
+    margin: 10px;
+    padding: 10px;
+    width: 400px;
```

Comment:
width: 400px is static
it means your content will not rendered properly in < 400px devices?